Integrating with your webapp/README.md

```diff
@@ -0,0 +1,167 @@
+# Integrate Anbox Cloud in your web application
+
+The dashboard included in Anbox Cloud is great to get started but writing your own web application
+
+## 0. AMC
+
+`amc` is the main tool you'll be using to interact with Anbox Cloud.
+
+ > `amc` is a simple wrapper over `ams` HTTP API. Everything you can do with `amc` can also be
+ > done through the API 
+
+### Accessing AMC
+
+Depending on how you deployed Anbox Cloud, `amc` can be accessed in different ways
+
+**Anbox Cloud Appliance (AWS Marketplace):**
+```sh
+$ ssh ubuntu@<instance IP>
+$ amc help
+```
+
+**Juju based deployment:**
+```sh
+$ juju ssh ams/0
+$ amc help
+```
+
+## 1. Creating the application
+
+Create a directory and place a file named `manifest.yaml` in it.
+
+```sh
+$ mkdir my-android10-app
+$ touch my-android10-app/manifest.yaml
+```
+
+And copy the following in `manifest.yaml`:
+
+```yaml
+name: my-android10-app
+instance-type: a2.3
+```
+
+Then use `amc` to create the application:
+
+```sh
+$ amc application create ./my-android10-app
+```
+
+`amc` will take a few minutes to prepare the application (apply security patches, optimize image size, run hooks, etc).
+You can monitor progress by running `amc application list` or block until it is ready with `amc wait -c status=ready my-android10-app`
+
+
+## 2. Building a simple web app to stream your Android applications
+
+Create a simple webpage containing an element with the ID `anbox-stream`. This is where the video stream will be displayed:
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+    <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
+    <meta content="utf-8" http-equiv="encoding">
+    <title>Anbox Streaming SDK Example</title>
+</head>
+<body>
+    <div id="anbox-stream"></div>
+</body>
+```
+
+Then include the stream SDK:
+
+```sh
+$ git clone https://github.com/anbox-cloud/anbox-streaming-sdk.git
```

Comment:
Alternatively, as we only need the JS SDK, we could ask people to do a 

`curl https://raw.githubusercontent.com/anbox-cloud/anbox-streaming-sdk/master/js/anbox-stream-sdk.js`